### PR TITLE
[WFLY-19611]:Upgrade OpenSAML to 4.3.0

### PR DESCRIPTION
**Pull request by jimma**
https://issues.redhat.com/browse/WFLY-19611

### CODE REVIEW

**Review by bstansberry — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -563,7 +563,7 @@
         <version.org.keycloak.keycloak-saml-wildfly-subsystem>18.0.2</version.org.keycloak.keycloak-saml-wildfly-subsystem>
         <version.org.kohsuke.metainf-services>1.11</version.org.kohsuke.metainf-services>
         <version.org.lz4.lz4-java>1.8.0</version.org.lz4.lz4-java>
-        <version.org.opensaml.opensaml>4.2.0</version.org.opensaml.opensaml>
+        <version.org.opensaml.opensaml>4.3.0</version.org.opensaml.opensaml>
```
Comment (jamezp) [inline pom.xml:566]: Is there a reason not to use 4.3.3?

**Review by jamezp — CHANGES_REQUESTED** [review]

**Comment by jamezp** [comment]
@jimma Do we using anything listed in https://shibboleth.atlassian.net/wiki/spaces/IDP4/pages/1265631499/ReleaseNotes#Changes-to-Existing-Behavior?

File: pom.xml (hunk 1 above)
Comment (jimma) [reply pom.xml:566]: 4.3.0 is the version which is used in CXF 4.0.5.  4.3.3 is not tested in wss4j and cxf.

**Comment by jimma** [comment]
> @jimma Do we using anything listed in https://shibboleth.atlassian.net/wiki/spaces/IDP4/pages/1265631499/ReleaseNotes#Changes-to-Existing-Behavior?

I think 4.3.0 should be safe to upgrade as this is tested in both wss4j 3.0.3 and CXF 4.0.5 release.

**Review by jamezp — APPROVED** [review]

**Comment by jamezp** [comment]
Upstream is #18089

**Comment by jamezp** [comment]
Thank you @jimma

**Pull request merged**